tests/worker/test_interceptor.py

```diff
@@ -283,3 +283,42 @@ def pop_trace(name: str, filter: Optional[Callable[[Any], bool]] = None) -> Any:
 
         # Confirm no unexpected traces
         assert not interceptor_traces
+
+
+class WorkflowInstanceAccessInterceptor(Interceptor):
+    def workflow_interceptor_class(
+        self, input: WorkflowInterceptorClassInput
+    ) -> Optional[Type[WorkflowInboundInterceptor]]:
+        return WorkflowInstanceAccessInboundInterceptor
+
+
+class WorkflowInstanceAccessInboundInterceptor(WorkflowInboundInterceptor):
+    async def execute_workflow(self, input: ExecuteWorkflowInput) -> int:
```

Comment:
Can you confirm that the `workflow.instance()` is non-`None` when this interceptor starts? I read through the code and I suspect it is, just wanted to confirm.

Reply:
Does this do what you want? https://github.com/temporalio/sdk-python/pull/739/commits/0b8a0bd2c06b8c46f6b2ebe5c2783c999c3c7900

Reply:
Yes, thanks!